doc/classes/@GlobalScope.xml

```diff
@@ -90,7 +90,7 @@
 			<param index="0" name="from" type="float" />
 			<param index="1" name="to" type="float" />
 			<description>
-				Returns the difference between the two angles, in the range of [code][-PI, +PI][/code]. When [param from] and [param to] are opposite, returns [code]-PI[/code] if [param from] is smaller than [param to], or [code]PI[/code] otherwise.
+				Returns the difference between the two radians, in the range of [code][-PI, +PI][/code]. When [param from] and [param to] are opposite, returns [code]-PI[/code] if [param from] is smaller than [param to], or [code]PI[/code] otherwise.
```

Comment:
```suggestion
				Returns the difference between the two angles (in radians), in the range of [code][-PI, +PI][/code]. When [param from] and [param to] are opposite, returns [code]-PI[/code] if [param from] is smaller than [param to], or [code]PI[/code] otherwise.
```
This is wrong, angle is not a unit, "degrees" is, and imo the mention of `PI` makes it clear, but if this needs to be changed this is the option

Reply:
I see, yeah. it was unclear to me when using the function that it accepted radians, the following -PI to PI mention made me think that was the range of the return value.